go/libkb/saltpack.go

```diff
@@ -137,3 +137,10 @@ func BoxPublicKeyToKeybaseKID(k saltpack.BoxPublicKey) (ret keybase1.KID) {
 	p := k.ToKID()
 	return keybase1.KIDFromRawKey(p, KIDNaclDH)
 }
+
+func checkSaltpackBrand(b string) error {
+	if b != KeybaseSaltpackBrand {
+		return KeybaseSaltpackError{}
+	}
+	return nil
+}
```

Comment:
Does it always make sense to error out on brands other than Keybase? Is it possible that someone might want to change the brand name but remain compatible with Keybase? (Or is that something we would want to discourage?)